run_hs_metrics.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash -l
+
+#SBATCH -A ngi2016001
+#SBATCH -n 2
+#SBATCH -N 1
+#SBATCH -p core
+#SBATCH -J hs_metrics
+#SBATCH -t 6:00:00
+#SBATCH -o hs_metrics.%j.out
+
+#
+# Run CollectHsMtrics for all recalibrated BAM files in an analysis folder
+#
+
+BAITS=$1
+TARGETS=$2
+ANALYSISDIR=$3
+INBAM=$4
+METRICS=$5
+
+# output folder for metrics and logs 
+OUTDIR="${ANALYSISDIR}/results/Reports/HsMetrics"
```

Comment:
just a note that, for 2.7, the logs are under `OUTDIR="${ANALYSISDIR}/results/Reports/[SAMPLE]/HsMetrics"` so this needs to be updated when the pipeline is changed

Reply:
I'm not sure what you mean with this comment @b97pla. The OUTDIR should be independent of the pipeline output structure. Of course we can change the structure of the output if we want to, but I can't see why it is a must.